engines/ospray/utils.cpp

```diff
@@ -193,4 +194,76 @@ void addInstance(OSPModel rootModel, OSPModel modelToAdd,
     ospAddGeometry(rootModel, instance);
     ospRelease(instance);
 }
+
+namespace osphelper
+{
+void set(OSPObject obj, const char* id, const char* s)
+{
+    ospSetString(obj, id, s);
+}
+void set(OSPObject obj, const char* id, const std::string& s)
+{
+    ospSetString(obj, id, s.c_str());
+}
+void set(OSPObject obj, const char* id, float v)
+{
+    ospSet1f(obj, id, v);
+}
+void set(OSPObject obj, const char* id, double v)
+{
+    ospSet1f(obj, id, v);
+}
+void set(OSPObject obj, const char* id, bool v)
+{
+    ospSet1b(obj, id, v);
+}
+void set(OSPObject obj, const char* id, int32_t v)
+{
+    ospSet1i(obj, id, v);
+}
+void set(OSPObject obj, const char* id, uint32_t v)
+{
+    ospSet1i(obj, id, v);
+}
+void set(OSPObject obj, const char* id, size_t v)
+{
+    ospSet1i(obj, id, v);
+}
+void set(OSPObject obj, const char* id, const Vector2f& v)
+{
+    ospSet2f(obj, id, v.x, v.y);
```

Comment:
Is it faster/better to use `ospSet2fv(obj, id, glm::value_ptr(v));`

Reply:
yes and yes